project_transportonly/project.json5

```diff
@@ -0,0 +1,354 @@
+                                'Truck_Medium+Regional+BEV_300',
+                                'Truck_Medium_Vocational+Local+BEV_150',
+                                'Truck_Medium_Vocational+Local+BEV_300',
+                                'Truck_Medium_Vocational+Regional+BEV_150',
+                                'Truck_Medium_Vocational+Regional+BEV_300',
+                                'Bus+N/A+BEV_200',
+                            ],
+                        },
+                        metric: {
+                            base: [
+                                'electricity_ev_mhdv_depot_ac', 'electricity_ev_mhdv_depot_dc',
+                                'electricity_ev_mhdv_opportunity_dc',
+                                'electricity_ev_mhdv_enroute_dc',
+                            ],
+                        },
+                        geography: {
+                            base: ['__all__'],
+                        }
+                    },
+                    {
+                        subsector: {
+                            base: ['Transit_Rail'],
+                        },
+                        metric: {
+                            base: ['electricity_rail_transit'],
+                        },
+                        geography: {
+                            base: ["04013", "04019", "05119", "06001", "06013", "06037", "06067", "06073", "06075", "06081", "06085", "08001", "08005", "08031", "08035", "08059", "09001", "09009", "10003", "11001", "12057", "12086", "13063", "13089", "13121", "17031", "17163", "17197", "18089", "18091", "18127", "18141", "22071", "24003", "24005", "24015", "24021", "24025", "24027", "24031", "24033", "24043", "24510", "25001", "25005", "25009", "25017", "25021", "25023", "25025", "25027", "26163", "27003", "27053", "27123", "27141", "29095", "29189", "29510", "34001", "34003", "34005", "34007", "34013", "34017", "34019", "34021", "34023", "34025", "34027", "34029", "34031", "34035", "34039", "34041", "36005", "36027", "36029", "36047", "36059", "36061", "36071", "36079", "36081", "36085", "36087", "36103", "36119", "37119", "39035", "39061", "40109", "41005", "41051", "41067", "42003", "42017", "42029", "42043", "42045", "42071", "42091", "42101", "44003", "44007", "44009", "47157", "48085", "48113", "48121", "48141", "48201", "48439", "49011", "49035", "49049", "49057", "51013", "51059", "51107", "51510", "51710", "53033", "53053", "53061", "54003", "54037", "55059", "55079"],
+                        },
+                    },
+                ],
+            },
+        },
+    ],
+
+    dimensions: {
+        base_dimensions: [
+            {
+                type: 'geography',
+                class: 'County',
+                name: 'US Counties 2020 L48',
+                display_name: 'County',
+                description: 'US counties for L48, Census 2020 vintage, no territories',
+                file: 'dimensions/counties.csv',
```

Comment:
This will likely eventually be very problematic. Counties come and go. We should strive for a minimal number of lists. This records will get de-duplicated at registration time. However, if a county gets added or removed, this copy may not get updated and will cause a hard-to-find bug. Can you point to the file paths in the main project for all duplicate record files?

Reply:
yes. I also pointed all other duplicates, dimensions, supplemental dimensions and supplemental dimension mappings, to ../project/xyz
I also added a supplemental dimension for the simplified vehicle types.